bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -410,10 +453,9 @@ def _canJsDisplay(self):
                     return False
         return True
 
-
     def _getJsCode(self):
-        # Workaround to have ConvertToJSON work
-        json = ROOT.TBufferJSON.ConvertToJSON(self.drawableObject, 3)
+        # produce JSON for the canvas
+        json = produceCanvasJsCode(self.drawableObject)
```

Comment:
Perhaps better `produceCanvasJson` ? Since JsCode we use it to refer to JavaScript.